heartbeat/docker-compose

```diff
@@ -113,26 +120,15 @@ END
 exit $OCF_SUCCESS
 }
 
-if [ -n "$OCF_RESKEY_binpath" ]; then
+if [ -r "$OCF_RESKEY_binpath" -x "$OCF_RESKEY_binpath" ]; then
```

Comment:
I think you meant to do `if [ -r "$OCF_RESKEY_binpath" ] && [ -x "$OCF_RESKEY_binpath" ]; then` here.